{
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unknown",
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "In Grafana 2.x through 6.x before 6.3.4, parts of the HTTP API allow unauthenticated use. This makes it possible to run a denial of service attack against the server running Grafana."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2022-01-13T00:00:00",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "name": "https://community.grafana.com/t/release-notes-v6-3-x/19202",
                    "url": "https://community.grafana.com/t/release-notes-v6-3-x/19202",
                    "x_refsource": "MISC"
                },
                {
                    "name": "https://github.com/grafana/grafana/releases",
                    "url": "https://github.com/grafana/grafana/releases",
                    "x_refsource": "MISC"
                },
                {
                    "name": "https://community.grafana.com/t/grafana-5-4-5-and-6-3-4-security-update/20569",
                    "url": "https://community.grafana.com/t/grafana-5-4-5-and-6-3-4-security-update/20569",
                    "x_refsource": "CONFIRM"
                },
                {
                    "name": "https://grafana.com/blog/2019/08/29/grafana-5.4.5-and-6.3.4-released-with-important-security-fix/",
                    "url": "https://grafana.com/blog/2019/08/29/grafana-5.4.5-and-6.3.4-released-with-important-security-fix/",
                    "x_refsource": "CONFIRM"
                },
                {
                    "name": "FEDORA-2019-0bb6b876da",
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/RF5ARGYX3WYB7H2FDR7VAWTEQ27UX3FU/",
                    "x_refsource": "FEDORA"
                },
                {
                    "name": "FEDORA-2019-77d612eab4",
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/UO4NBL7PKW4OSFRVZENGC42EWEJV2YAH/",
                    "x_refsource": "FEDORA"
                },
                {
                    "name": "https://security.netapp.com/advisory/ntap-20191004-0004/",
                    "url": "https://security.netapp.com/advisory/ntap-20191004-0004/",
                    "x_refsource": "CONFIRM"
                },
                {
                    "name": "openSUSE-SU-2020:0892",
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2020-06/msg00060.html",
                    "x_refsource": "SUSE"
                },
                {
                    "name": "openSUSE-SU-2020:1105",
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2020-07/msg00083.html",
                    "x_refsource": "SUSE"
                },
                {
                    "name": "openSUSE-SU-2020:1611",
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2020-10/msg00009.html",
                    "x_refsource": "SUSE"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2019-15043",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "In Grafana 2.x through 6.x before 6.3.4, parts of the HTTP API allow unauthenticated use. This makes it possible to run a denial of service attack against the server running Grafana."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://community.grafana.com/t/release-notes-v6-3-x/19202",
                            "refsource": "MISC",
                            "url": "https://community.grafana.com/t/release-notes-v6-3-x/19202"
                        },
                        {
                            "name": "https://github.com/grafana/grafana/releases",
                            "refsource": "MISC",
                            "url": "https://github.com/grafana/grafana/releases"
                        },
                        {
                            "name": "https://community.grafana.com/t/grafana-5-4-5-and-6-3-4-security-update/20569",
                            "refsource": "CONFIRM",
                            "url": "https://community.grafana.com/t/grafana-5-4-5-and-6-3-4-security-update/20569"
                        },
                        {
                            "name": "https://grafana.com/blog/2019/08/29/grafana-5.4.5-and-6.3.4-released-with-important-security-fix/",
                            "refsource": "CONFIRM",
                            "url": "https://grafana.com/blog/2019/08/29/grafana-5.4.5-and-6.3.4-released-with-important-security-fix/"
                        },
                        {
                            "name": "FEDORA-2019-0bb6b876da",
                            "refsource": "FEDORA",
                            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/RF5ARGYX3WYB7H2FDR7VAWTEQ27UX3FU/"
                        },
                        {
                            "name": "FEDORA-2019-77d612eab4",
                            "refsource": "FEDORA",
                            "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/UO4NBL7PKW4OSFRVZENGC42EWEJV2YAH/"
                        },
                        {
                            "name": "https://security.netapp.com/advisory/ntap-20191004-0004/",
                            "refsource": "CONFIRM",
                            "url": "https://security.netapp.com/advisory/ntap-20191004-0004/"
                        },
                        {
                            "name": "openSUSE-SU-2020:0892",
                            "refsource": "SUSE",
                            "url": "http://lists.opensuse.org/opensuse-security-announce/2020-06/msg00060.html"
                        },
                        {
                            "name": "openSUSE-SU-2020:1105",
                            "refsource": "SUSE",
                            "url": "http://lists.opensuse.org/opensuse-security-announce/2020-07/msg00083.html"
                        },
                        {
                            "name": "openSUSE-SU-2020:1611",
                            "refsource": "SUSE",
                            "url": "http://lists.opensuse.org/opensuse-security-announce/2020-10/msg00009.html"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2019-15043",
        "dateUpdated": "2022-01-13T00:00:00",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}